google/resource_bigquery_dataset_test.go

```diff
@@ -205,7 +263,7 @@ resource "google_bigquery_dataset" "test" {
   description                 = "This is a bar description"
   location                    = "EU"
   default_partition_expiration_ms = 7200000
-  default_table_expiration_ms = 7200000
+	default_table_expiration_ms = 7200000
```

Comment:
nit: this line is a tab, but should be spaces for consistency with other test configs

Reply:
whoops. fixed.